3rdparty/CMakeLists.txt

```diff
@@ -275,6 +275,10 @@ add_library(spirv_cross OBJECT
 )
 target_compile_definitions(spirv_cross PUBLIC SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS)
 
+# cmake-git-version-tracking
+add_subdirectory(git-version-tracking EXCLUDE_FROM_ALL)
```

Comment:
wont this just produce a `git hash` info of the submodule and not nabla parent repo itself?

Reply:
back then probably, now Tadzio fixed it and with the [tool ](https://github.com/Devsh-Graphics-Programming/Nabla/pull/741#issuecomment-2338841799) we control it per single invocation